code/src/java/pcgen/core/analysis/SkillModifier.java

```diff
@@ -34,6 +35,10 @@
 public final class SkillModifier
 {
 
+	private SkillModifier()
```

Comment:
Why delete private constructor?  Isn't that meaningful for a class designed not to be constructed?